next-tavla/app/(admin)/actions.ts

```diff
@@ -104,3 +104,14 @@ export async function getBoardsForUser() {
         )
         .flat()
 }
+
+export async function getBoardsForUser() {
+    const user = await getUserFromSessionCookie()
+    if (!user) return redirect('/')
+
+    const boardRef = await firestore().collection('boards').get()
```

Comment:
Her tror jeg vi må gjøre noe mer for å kun hente boards til en spesifikk bruker. kanksje ta inn uid også filtrere boards owners på det elns 💭